src/Umbraco.PublishedCache.HybridCache/SeedKeyProviders/Document/DocumentBreadthFirstKeyProvider.cs

```diff
@@ -6,10 +6,74 @@ namespace Umbraco.Cms.Infrastructure.HybridCache.SeedKeyProviders.Document;
 
 internal sealed class DocumentBreadthFirstKeyProvider : BreadthFirstKeyProvider, IDocumentSeedKeyProvider
 {
+    private readonly IDocumentNavigationQueryService _navigationQueryService;
```

Comment:
Looks like this is already populated on the base class, but as a private field.  So maybe it's better to upgrade that on the base class to a protected property, so you can access it from this derived class?